src/modules/logger/logger.service.ts

```diff
@@ -52,6 +52,10 @@ export class Logger extends NestLogger implements LoggerService {
     });
   }
 
+  setContext(ctx: string) {
+    this.context = ctx;
+  }
```

Comment:
Why this change?

Reply:
after migrating to nest js v8.0.0-alpha(not sure if thats the reason), I started getting this kinda error in all services where Logger is used

![image](https://user-images.githubusercontent.com/23248910/124720721-7da3df00-df19-11eb-80f5-dd80aa0a0150.png)

I added this method to sets the context of Logger, accepting context name.

it was missing and can't figure out how it was working before